packages/ckeditor5-theme-lark/theme/ckeditor5-link/link.css

```diff
@@ -7,3 +7,18 @@
 .ck .ck-link_selected {
 	background: var(--ck-color-link-selected-background);
 }
+
+/* Classes for fake visual selection displayed on text when LinkUI has focus. */
+.ck .ck-link_fake-selection {
+	background: var(--ck-color-link-fake-selection);
+}
+
+/* Collapsed fake visual selection. */
+.ck .ck-link_fake-selection_collapsed {
+	display: inline-block;
```

Comment:
Are you sure about this style? ATM it looks like

![2020-06-16 10 00 07](https://user-images.githubusercontent.com/1099479/84748030-60237080-afb8-11ea-8561-26520267f823.gif)

but with 

```css
display: inline;
line-height: 100%;
```

it could be more like a real caret (mostly talking about the height):

![2020-06-16 10 00 24](https://user-images.githubusercontent.com/1099479/84748058-687bab80-afb8-11ea-8246-f567f96fac2c.gif)

Reply:
Yeah, you're right. I had some problems with inline - but I don't recall why exactly. Now it looks better.